R/DataLoadingSaving.R

```diff
@@ -118,32 +118,24 @@ getDbCohortMethodData <- function(connectionDetails,
                                   outcomeTable = "condition_occurrence",
                                   cdmVersion = "5",
                                   firstExposureOnly = FALSE,
-                                  removeDuplicateSubjects = FALSE,
+                                  removeDuplicateSubjects = "keep all",
                                   restrictToCommonPeriod = FALSE,
                                   washoutPeriod = 0,
                                   maxCohortSize = 0,
                                   covariateSettings) {
-  if (is.null(studyStartDate)) {
-    studyStartDate <- ""
-  }
-  if (is.null(studyEndDate)) {
-    studyEndDate <- ""
-  }
-  if (is.logical(removeDuplicateSubjects)) {
-    if (removeDuplicateSubjects) {
-      removeDuplicateSubjects <- "remove all"
-    } else {
-      removeDuplicateSubjects <- "keep all"
-    }
-  }
   errorMessages <- checkmate::makeAssertCollection()
```

Comment:
Now that DatabaseConnector 6 has been released we can require that version, and just use `checkmate::assertClass(connectionDetails, "ConnectionDetails", add = errorMessages)`

Reply:
Done

Reply:
[ae4a5cf](https://github.com/OHDSI/CohortMethod/pull/136/commits/ae4a5cff840968a4c8f7d47c84cc2d0c81bd4832)